src/bindings/python/src/pyopenvino/core/offline_transformations.cpp

```diff
@@ -132,12 +132,14 @@ void regmodule_offline_transformations(py::module m) {
 
     m_offline_transformations.def(
         "paged_attention_transformation",
-        [](std::shared_ptr<ov::Model> model) {
+        [](std::shared_ptr<ov::Model> model, bool use_block_indices_inputs, bool use_score_outputs) {
             ov::pass::Manager manager;
-            manager.register_pass<ov::pass::SDPAToPagedAttention>();
+            manager.register_pass<ov::pass::SDPAToPagedAttention>(use_block_indices_inputs, use_score_outputs);
```

Comment:
We use `block_indices_input` even without this PR. Am I right that letter `s` in `use_block_indices_inputs` should tell us that this argument adds _multiple_ inputs instead of just _one_ input?

```suggestion
            manager.register_pass<ov::pass::SDPAToPagedAttention>(add_block_indices_input_per_layer, add_score_outputs);
```